main.go

```diff
@@ -75,11 +77,16 @@ func main() {
 	ctx := context.Background()
 	log := logger.NewLogger(ctx)
 	ctx = context.WithValue(ctx, logger.LoggerKey{}, log)
+	undo, maxprocsError := maxprocs.Set(maxprocs.Logger(stdlog.Printf))
```

Comment:
Please use the existing logger implementation at pkg/logger

Reply:
Sure